.github/workflows/validate-documentation.yml

```diff
@@ -16,7 +16,13 @@ jobs:
 
     steps:
     - name: check out code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v3.1.0
+
+    - name: verify links by lycheeverse
+      uses: lycheeverse/lychee-action@v1.5.1
+      with:
+        fail: true
+        args: "--threads 1 --max-concurrency 1 --verbose --no-progress './**/*.md' './**/*.html'"
```

Comment:
In this repository there is tons of links to the GitHub. It leads to throttling. One of the way is to use once request at a time.